### DISCUSSION TITLE
Ubuntu Desktop

### DISCUSSION THREAD

**Discussion by Gotcha007** [RaspAP/raspap-webgui#807] [Q&A]
Hi guys, is there a way to install it on a RPI 4 Ubuntu 64 desktop edition?
I know it's not support but wondering if there was a workaround for this.

**Comment by billz** [comment]
Have you tried the manual setup instructions? Follow the steps, see what breaks and find a workaround. This is the process that's used to extend support to other OS's. https://docs.raspap.com/manual/

Note: Bridged AP mode is currently unsupported on Ubuntu. See #537
